package.json

```diff
@@ -35,7 +35,7 @@
     "bunyan": "1.2.0",
     "compute-cluster": "0.0.9",
     "convict": "0.6",
-    "fxa-auth-mailer": "1.0.7",
+    "fxa-auth-mailer": "git+https://github.com/mozilla/fxa-auth-mailer.git#a7633712b",
```

Comment:
I hard coded these URLs until @dannycoates returns and can release new versions of each.